components/eamxx/src/share/io/scorpio_output.cpp

```diff
@@ -196,20 +254,36 @@ void AtmosphereOutput::run (const std::string& filename, const bool is_write_ste
   }
 
   // If needed, remap fields from their grid to the unique grid, for I/O
-  if (m_remapper) {
-    start_timer("EAMxx::IO::remap");
-    m_remapper->remap(true);
+  if (m_vert_remapper) {
```

Comment:
Would it be better/cleaner to add a `map` to scorpio_output.cpp that stores all remappers in a specific order and calls them in a loop?  I ask because this chunk of code matches the horiz_remapper one below, just with a different remapper.

Reply:
You can turn the body of these if statements into a lambda, and call the lambda passing the two remappers. I don't think we need a map for remappers.

Reply:
oh good idea.